### PR TITLE
Update cloud-provider-openstack testgrid config

### PR DESCRIPTION
**Pull request by mrhillsman**
Add v1.11 conformance

Signed-off-by: Melvin Hillsman <mrhillsman@gmail.com>

### CODE REVIEW

File: testgrid/config.yaml [hunk 1]
```diff
@@ -3612,6 +3633,9 @@ dashboards:
   - name: "Conformance - GCE"
     description: Runs conformance tests using kubetest against latest kubernetes release 1.11 branch on GCE
     test_group_name: ci-kubernetes-gce-conformance-latest-1-11
+  - name: "Conformance - OpenStack"
+    description: Runs conformance tests using kubetest against kubernetes v1.11 with cloud-provider-openstack
+    test_group_name: ci-periodic-cloud-provider-openstack-acceptance-test-e2e-conformance-stable-branch-v1.11
   # TODO(aishsundar) (Add 1.11 OpenStack Conformance results to release dashboard)
```
Comment (kiwik) [inline testgrid/config.yaml]: Please remove TODO, we have got it now :) and we should add this config into **sig-release-1.11-blocking** section, see https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml#L3717

**Comment by dims** [comment]
cc @AishSundar

**Comment by dims** [comment]
Thanks Sen! @krzyzacy

**Pull request merged**